Sources/FoundationInternationalization/TimeZone/TimeZone_Cache.swift

```diff
@@ -106,6 +110,23 @@ struct TimeZoneCache : Sendable {
                 }
             }
 
+#if os(Windows)
+            let hFile = _TimeZone.TZDEFAULT.withCString(encodedAs: UTF16.self) {
+                CreateFileW($0, GENERIC_READ, DWORD(FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE), nil, DWORD(OPEN_EXISTING), 0, nil)
```

Comment:
Do we need all these permissions? Seems like `FILE_SHARE_READ` should suffice?

Reply:
They are not file permissions - the permission being requested is `GENERIC_READ`.  The bits that you are referencing (`FILE_SHARE_*`) are file open disposition flags.  They are all needed to permit other processes to simultaneously create handles to the file with read/write/delete access.